invenio_rdm_records/services/pids/providers/external.py

```diff
@@ -41,9 +41,8 @@ def __call__(self, record, identifier, provider, errors):
         for p in self.prefixes:
             if identifier.startswith(p):
                 errors.append(
-                    _("The prefix '{prefix}' is administrated locally.").format(
-                        prefix=p
-                    )
+                    _("The prefix '{prefix}' is managed by Zenodo.").format(prefix=p)
```

Comment:
I should remove the reference to Zenodo in favour of "managed locally" at least. Is it possible to access the site name from the config here?

Reply:
:+1: You should be able to do something like ``current_app.config['THEME_SITENAME']``

Reply:
Tested and works perfectly